overlord/snapshotstate/backend/backend.go

```diff
@@ -857,11 +924,36 @@ func (se *SnapshotExport) Close() {
 	se.snapshotFiles = nil
 }
 
+type earlyJSON struct {
+	ContentHash []byte `json:"content-hash"`
+}
+
 func (se *SnapshotExport) StreamTo(w io.Writer) error {
 	// write out a tar
 	var files []string
 	tw := tar.NewWriter(w)
 	defer tw.Close()
+
+	// export contentHash as early.json
+	h, err := json.Marshal(earlyJSON{se.contentHash})
+	if err != nil {
+		return err
+	}
+	hdr := &tar.Header{
+		Typeflag: tar.TypeReg,
+		Name:     "early.json",
```

Comment:
content.json perhaps?  another possible name if we want to convey that it comes early would be preview.json